hw/ip/prim/lint/prim_assert.waiver

```diff
@@ -13,3 +13,5 @@ waive -rules {LINE_LENGTH} -location {prim_assert.sv} -msg {Line length of} \
       -comment "Some macros cannot be line-wrapped, as some tools do not support that."
 waive -rules {LINE_LENGTH} -location {prim_flop_macros.sv} -msg {Line length of} \
       -comment "Some macros cannot be line-wrapped, as some tools do not support that."
+waive -rules {LINE_LENGTH} -location {prim_assert_sec_cm.svh} -msg {Line length of} \
+      -comment "Some macros cannot be line-wrapped, as some tools do not support that."
```

Comment:
Maybe a sign that we need to think a bit harder about the super verbose names :-)

Reply:
yeah...